pkgs/misc/vscode-extensions/update_installed_exts.sh

```diff
@@ -0,0 +1,38 @@
+#! /usr/bin/env bash
+
+CODE=$1
+
+if [ -z "$CODE" ]; then
+  CODE=$(pwd)/result/bin/code
+fi
+
+function get_ver() {
+  curl --silent "https://raw.githubusercontent.com/$1/$2/master/package.json" | grep '"version":' | tr -d '[:alpha:]", :'
+}
+
+function get_vsixpackage_sha() {
+  URL="https://$1.gallery.vsassets.io/_apis/public/gallery/publisher/$1/extension/$2/latest/assetbyname/Microsoft.VisualStudio.Services.VSIXPackage"
+  curl --silent "$URL" | sha256sum | awk '{ print $1 }'
+}
+
+printf '{ extensions = [\n'
+for i in $($CODE --list-extensions)
+do
+  OWNER=$(echo $i | awk -F '.' '{ print $1 }';)
+  EXT=$(echo $i | awk -F '.' '{ print $2 }';)
+
+  SHA=$(get_vsixpackage_sha $OWNER $EXT)
+
+  VER=$(get_ver $OWNER $EXT)
+
+  if [ -z "$VER" ]; then
+    EXT2="vscode-$(echo $EXT | tr '[:upper:]' '[:lower:]')"
+    VER=$(get_ver $OWNER $EXT2)
+  fi
+  if [ -z "$VER" ]; then
+    VER="unknown"
```

Comment:
Should the script not rather fail in the case the version could not be retrieved?